src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/ccd/Event.java

```diff
@@ -55,9 +55,9 @@
     SUBMIT_REASONS_FOR_APPEAL("submitReasonsForAppeal"),
     REQUEST_CLARIFYING_ANSWERS("requestClarifyingAnswers"),
     SUBMIT_CLARIFYING_ANSWERS("submitClarifyingAnswers"),
+    EDIT_DOCS("editDocs"),
```

Comment:
hehe, no worries, yes, you sound pedantic ;)  I will change it, yes, I fell into this habit because in CCD there is a limit of characters for the name of the event. But yeah, editDocuments is not that long anyway. 
Thanks

Reply:
TODO

Reply:
done